x-pack/plugins/cases/common/api/cases/comment/index.test.ts

```diff
@@ -340,6 +341,23 @@ describe('Comments', () => {
         right: defaultRequest,
       });
     });
+
+    it.skip('throws error when comment is too long', () => {
```

Comment:
@cnasikas  This test doesn't pass because `CommentRequestRt ` is a huge object containing all other comment types as well. it is hard to verify.

Reply:
Thanks Christos!! Resolved with `PathReporter `